api/actions/views.py

```diff
@@ -65,7 +73,7 @@ class ActionDetail(JSONAPIBaseView, generics.RetrieveAPIView):
     permission_classes = (
         permissions.IsAuthenticatedOrReadOnly,
         base_permissions.TokenHasScope,
-        ReviewActionPermission,
```

Comment:
It broke `RequestActions` going through the same endpoint. IIRC, I did some object permission magic to avoid breaking any of the tests for `ReviewActions`, but I'll double check that